largest_palindrome_product.js

```diff
@@ -5,18 +5,42 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function(digits) {
   var factor_0 = 0;
   var factor_1 = 0;
   var palindromeNumber = 0;
 
   // do your work here
 
+  function checkPalindrome(num) {
+    let numToString = num.toString();
+    const numOfDigits = numToString.length;
+    for (let i = 0; i < Math.floor(numOfDigits / 2); i++) {
```

Comment:
Insert newline above the for loop

Reply:
Instead of using a for loop, you already have the number as a string with `numToString`. You could always use something like this to check for a palindrome:
```
const reverse = numToString.split('').reverse().join('');
return numToString === reverse;
```